app/assets/stylesheets/conversations.scss

```diff
@@ -180,6 +180,17 @@
     a:hover {
       text-decoration: none;
     }
+    .pagination {
+      margin-left: auto;
+      margin-right: auto;
+      text-align: center;
+    }
+  }
+}
+
+@media (min-width: 1024px) and (max-width: 1354px) {
```

Comment:
I think the `min-width` is useless here. That would make the padding bigger if the window width is smaller than 1024px. Just define a max-width.